src/DotNetWorker.Core/DotNetWorker.Core.csproj

```diff
@@ -22,12 +22,27 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Core" Version="1.37.0" />
-    <PackageReference Include="Microsoft.Extensions.Hosting" Version="5.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="5.0.0" />
-    <PackageReference Include="System.Collections.Immutable" Version="5.0.0" />
-    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="7.0.0" />
   </ItemGroup>
 
+  <Choose>
+    <When Condition="'$(TargetFramework)' == 'net9.0'">
+      <ItemGroup>
+        <PackageReference Include="Microsoft.Extensions.Hosting" Version="9.0.0-preview.*" />
+        <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="9.0.0-preview.*" />
+        <PackageReference Include="System.Collections.Immutable" Version="9.0.0-preview.*" />
+        <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="9.0.0-preview.*" />
```

Comment:
I think this being `preview.*` makes sense, but we don't typically do wildcards here, right? Just tactics for preview?

Reply:
Correct. We usually ping to a specific version. We can change to that later.